### ISSUE TITLE
Horses have become invulnerable in Development and Commit builds

### ISSUE THREAD

**Issue by PaulFromNZ** [Carbon-Modding/Carbon.Core#46]
RidableHorses have become invulnerable.  They take no damage from guns, explosives, fire, melee and they do not deteriorate.
They do not eat ever again (once health and stamina are full), and they do not poop, ever.
This applies to horses whether they are hitched or not and whether console spawned, bought, wild or pre-existing.

This issue does NOT occur in Production (8dcf5e6) or Staging (3f9ddc1) builds.
This issue DOES occur with Development (aa3d978) and the last few Commit builds (including 8d3f714).

**Comment by raulssorban** [comment]
Hey there. We've found the issue. To temporarily fix this, please remove "assets/rust.ai/nextai/testridablehorse.prefab" from the RustEdit.Ext config under ManagedPrefabs. That seems to be an oversight as those entities are protected by the module from being damaged.

**Comment by PaulFromNZ** [comment]
Ta, and strange.  I hadn't enabled the RustEdit module and I was scratching my head as the RustEdit.Ext config was practically empty.  Just "Enabled": false and "Config": {}. File size less than 1KB.
I enabled RustEdit True under build efafddb and it populated the config file with numerous entries, but without the entry you mentioned needed removing.
Tested horses can take damage again.  They can.
Deleted the RustEdit.Ext folder and allowed it to be regenerated on restart and this time "Enabled: False" was set by default, but it also populated all the entries (25KB size), again, without the horse prefab.
Tested with RustEdit off and horses can still take damage.  So it looks like the immortality issue is sorted in efafddb
Will now test that horses will eat and poop again, but that may take a little while.
